charts/INSTALL.md

```diff
@@ -0,0 +1,291 @@
+# OpenSDS Installation using Helm # 
+
+## Prerequisite ##
+
+### ubuntu
+* Version information
+
+	```
+	root@proxy:~# cat /etc/issue
+	Ubuntu 16.04.2 LTS \n \l
+	```
+
+
+#### Required Packages 
+
+* Install following packages:
+    ```bash
+    apt-get install socat gcc make libc-dev docker.io
+    apt-get install -y git curl wget libltdl7 libseccomp2
+    apt-get install python-pip
+    
+    ```
+* Install golang (Version 1.11.2)
+    ```bash
+    wget https://storage.googleapis.com/golang/go1.11.2.linux-amd64.tar.gz
+    tar -C /usr/local -xzf go1.11.2.linux-amd64.tar.gz   
+    echo 'export PATH=$PATH:/usr/local/go/bin' >> /etc/profile  
+    echo 'export GOPATH=$HOME/gopath' >> /etc/profile  
+    source /etc/profile
+    ```
+* Install etcd (Version 3.3.1)
+    ```bash
+    curl -L https://github.com/coreos/etcd/releases/download/v3.3.1/etcd-v3.3.1-linux-amd64.tar.gz -o etcd-v3.3.1-linux-amd64.tar.gz
+    tar xzvf etcd-v3.3.1-linux-amd64.tar.gz
+    cd etcd-v3.3.1-linux-amd64
+    sudo cp etcd /usr/local/bin/
+    sudo cp etcdctl /usr/local/bin/
+    etcd –-version
+
+    ```
+* Install docker-compose:
+
+    ```
+    curl -L "https://github.com/docker/compose/releases/download/1.23.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    chmod +x /usr/local/bin/docker-compose
+    ```
+### [kubernetes](https://github.com/kubernetes/kubernetes) local cluster
+* You can startup `v1.13.0` k8s local cluster by executing commands below, etcd version 3.2.24 or greater required:
+
+	```
+	cd $HOME
+	git clone https://github.com/kubernetes/kubernetes.git
+	cd $HOME/kubernetes
+	git checkout v1.13.0
+	make
+	echo alias kubectl='$HOME/kubernetes/cluster/kubectl.sh' >> /etc/profile
+	ALLOW_PRIVILEGED=true FEATURE_GATES=CSIPersistentVolume=true,MountPropagation=true,VolumeSnapshotDataSource=true,KubeletPluginsWatcher=true RUNTIME_CONFIG="storage.k8s.io/v1alpha1=true" LOG_LEVEL=5 hack/local-up-cluster.sh
```

Comment:
I think `CSIPersistentVolume`, `VolumeSnapshotDataSource` feature gates and `RUNTIME_CONFIG` flag can be removed in k8s v1.13. You can change the command to `ENABLE_DAEMON=true FEATURE_GATES=MountPropagation=true LOG_LEVEL=5 hack/local-up-cluster.sh -O`